src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature_pb2.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature.proto
```

Comment:
Protos should be compiled from src and the path should ostorlab/agent/message/proto/v3/fingerprint/file/ios/feature/feature.proto